### ISSUE TITLE
Get List Of Current User's Playlists Not Working

### ISSUE THREAD

**Issue by seijimorimoto** [spotify/web-api#1363]
Issue found on October 18th 2019.

#### Endpoint(s):
* `GET /v1/me/playlists`

#### Scope(s):
* None (application is not using authentication)

#### Steps to reproduce:
1. Open the API Console
2. Call the `get-current-user-playlists` endpoint.

#### Expected behaviour:
An array of simplified playlist objects as described on `https://developer.spotify.com/documentation/web-api/reference/object-model/#playlist-object-simplified`.

#### Actual behaviour:
500 Server Error (consistently)

![image](https://user-images.githubusercontent.com/7454295/67093684-1f195900-f1aa-11e9-9470-afd1a1a927de.png)

**Comment by SuN-80** [comment]
I have the exact same problem, for a week or so.

Additionally none of my playlists appear on my profile anymore (trying to create one using the web interface even returns an unknown error), and if I try to open one with its URLs it says "upstream request timeout".

**Comment by kingosticks** [comment]
> Scope(s):
>
>   None (application is not using authentication)

You need to to authenticate the current user and have the necessary `playlist-read-private` scope. The API Console works fine for me.

**Comment by SuN-80** [comment]
My playlists are public. They are created using the appropriate rights, but anybody should be able to see the content. What do you see if you click the link I've put in my previous comment please?

I used to see the thousand playlists created, but it stopped working from one to day another. I still don't get it. Even our profile displays this error...

**Comment by kingosticks** [comment]
`/v1/me/playlists` needs to know the `me` part. It gets that's from the auth. 

I also get the same timeout with https://open.spotify.com/playlist/5tH3jwdbxfOpMaspvdpJ3H but that's not related to using the `/v1/me/playlists` endpoint and therefore not the same issue. 

I suggest you talk to Spotfy's regular user support, I doubt you'll get anything help from Spotify here.

**Comment by seijimorimoto** [comment]
> > Scope(s):
> > None (application is not using authentication)
> 
> You need to to authenticate the current user and have the necessary `playlist-read-private` scope. The API Console works fine for me.

The issue is now solved for me (actually it was automatically solved a couple of days later after my original post). However, I just wanted to say that I did authenticate myself by getting the OAuth Token using the API Console and I was still getting the problem back then. The issue was also not the missing `playlist-read-private` scope, since I had public playlists in my account. And even if I hadn't, I should have received an empty array and not a 500 server error, as specified in the Docs.

I don't know what could have been the problem, since it appeared out of nowhere and then it randomly vanished. Hope no one else experience it.

**Comment by kingosticks** [comment]
Considering your original post stated you didn't authenticate I think that was a little unclear. 

> The issue was also not the missing playlist-read-private scope, since I had public playlists in my account. And even if I hadn't, I should have received an empty array and not a 500 server error, as specified in the Docs.

Agreed, just another transient problem with Spotify's servers then.

**Comment by seijimorimoto** [comment]
My bad, I accidentally wrote the "not" when writing "application not using authentication" in my original post.

> Agreed, just another transient problem with Spotify's servers then.

Sure, probably it was just a temporary problem as you said.

**Comment by SuN-80** [comment]
> I also get the same timeout with https://open.spotify.com/playlist/5tH3jwdbxfOpMaspvdpJ3H but that's not related to using the /v1/me/playlists endpoint and therefore not the same issue.

yes it is. Here is what Spotify's console returns when I use this endpoint:
```
{
  "error": {
    "status": 500,
    "message": "Server error."
  }
}
```
It started out of nowhere and I don't know how to fix this.

> 
I suggest you talk to Spotfy's regular user support about the issue you have with your account, I doubt you'll get anything help from Spotify here.

We replied here because we did not get any reply from Spotify...
